cuppa/src/test/java/org/forgerock/cuppa/ModelTests.java

```diff
@@ -97,9 +102,34 @@ public void testShouldTakeDefensiveCopiesOfMutableObjects() {
         assertThat(test.options.get(TestOption.class)).isEmpty();
     }
 
+
+    @Test
+    public void shouldBothTestsOfMatchingTags() throws Exception {
```

Comment:
Not sure this test name makes sense. Perhaps instead `shouldMergeMultipleOptionsOfSameType`?